oic.r.tokenrefresh.raml

```diff
@@ -0,0 +1,50 @@
+#%RAML 0.8
+title: Account Token Refresh
+version: v1.0-20180320
+documentation:
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+schemas:
+- TokenRefresh-Request: !include schemas/oic.r.tokenrefresh-request.json
+- TokenRefresh-Response: !include schemas/oic.r.tokenrefresh-response.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.baseline"]
+          
+/oic/sec/tokenrefresh:
+  description: |
+    Resource to manage the access-token using refresh token
+
+  post:
+    description: |
+      Obtain fresh access-token using the refresh token, client should refresh access-token before it expires.
+    body:
+      application/json:
+        schema: TokenRefresh-Request
+        example: |
+          {
+            "uid" : "123e4567-e89b-12d3-a456-d6e313b71d9f",
+            "di" : "9cfbeb8e-5a1e-4d1c-9d01-00c04fd430c8",
+            "refreshtoken" : "00fe4644a6fbe5324eec"
+          }
+    responses:
+      204:
+        description: |
+          2.04 Changed respond with new access-token
+        body:
+          application/json:
+            schema: TokenRefresh-Response
+            example: |
+              {
+                "accesstoken" : "8ce598980761869837be",
+                "refreshtoken" : "d4922312b6df0518e146",
+                "expiresin" : 3600
+              }
```

Comment:
Add expected response to RETRIEVE request to align with:

The RETRIEVE operation on OCF Cloud’s “/oic/sec/ tokenrefresh” Resource is not allowed and the OCF Cloud is expected to reject all attempts to perform such operation.

Reply:
Currently "get" is not defined in this schema, which should mean this operation is not supported. @marktrayer , can you also please provide inputs on this, if we need to add anything.